app/move/views/view.njk

```diff
@@ -60,12 +60,53 @@
   </header>
 {% endblock %}
 
+{% set urls = {
```

Comment:
I think my concern is still defining maps like this within a view. If we need logic to loop over things and set properties based on certain criteria that kind of logic should live in a piece of middleware/controller where it can be unit tested.

Then you just pass a simple object/array into a component or loop over the items of the object/array in the view.

Reply:
Not sure why there are 2 separate comment threads on this line - but, as noted above, moved to the view controller